lib/builders/MaterialBuilder.ts

```diff
@@ -73,7 +73,12 @@ export class MaterialBuilder {
    * Outputs a static theme stylesheet, based on the specified options
    */
   _buildTheme(buildData: MaterialToolsData): MaterialToolsOutput {
-    if (!this._themeBuilder) {
+
+    // When the ThemeBuilder is not initialized and theme definitions are specified.
+    if (!this._themeBuilder && this._themes) {
+      let moduleName = path.join(buildData.package.module, this._options.mainFilename);

```

Comment:
As discussed we are moving that logic into a method, because we do not want to depend on `getData`